app/Jobs/GenerateSimilarityIndex.php

```diff
@@ -78,8 +79,11 @@ public function __construct(Volume $volume)
      */
     public function handle()
     {
-       $imageHashArray = $this->createImageHashArray();
-        if ($imageHashArray !== false) {
+        $collection = collect($this->volume->images);
```

Comment:
This already is a collection.
```suggestion
        $collection = $this->volume->images;
```